src/bootstrap/test.rs

```diff
@@ -1131,7 +1131,6 @@ note: if you're sure you want to do this, please open an issue as to why. In the
             }
         }
         flags.push(format!("-Cdebuginfo={}", builder.config.rust_debuginfo_level_tests));
-        flags.push("-Zunstable-options".to_string());
```

Comment:
Why was this changed? It seems like the most likely candidate for the CI failing.

Reply:
See the PR description:

> To get the test working I had to remove -Z unstable-options, which x.py passed to compiletest unconditionally. It was first added in 8c2ec68 so -Z miri would be allowed. -Z miri is no longer passed unconditionally, so hopefully removing it won't break anything.

That caused a lot more breakage than I expected so I might remove the test for now :/ or wait to merge this until I can make a separate PR removing `unstable-options`.

Reply:
> That caused a lot more breakage than I expected so I might remove the test for now :/ or wait to merge this until I can make a separate PR removing unstable-options.

I removed the test for now.